pkg/tests/stamp_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test time stamping in pkg_tar"""
+
+import tarfile
+import time
+import unittest
+
+from bazel_tools.tools.python.runfiles import runfiles
+
+PORTABLE_MTIME = 946684800  # 2000-01-01 00:00:00.000 UTC
+
+class PkgTarTest(unittest.TestCase):
+  """Testing for pkg_tar rule."""
+
+  def assertTarFilesAreAlmostNew(self, file_name):
+    """Assert that tarfile contains files with an mtime of roughly now.
+
+    Args:
+        file_name: the path to the TAR file to test.
+    """
+    file_path = runfiles.Create().Rlocation('rules_pkg/tests/' + file_name)
```

Comment:
Nit: consider `os.environ['TEST_WORKSPACE']` instead of `rules_pkg`, is more portable than calling out `rules_pkg`.